its/ruling/src/test/resources/expected/php-S1481.json

```diff
@@ -164,6 +160,9 @@
 323,
 325,
 ],
+'project:PHPExcel_1.7.9_doc/Classes/PHPExcel/Shared/JAMA/Matrix.php':[
+896,
```

Comment:
@pynicolas It's not easy to fix this FP now. And as it anyway points out some problem in code we decided to keep it. This ticket should solve that kind of issues (when used undeclared in this file symbol) https://jira.sonarsource.com/browse/SONARPHP-588